contracts/token/BasicToken.sol

```diff
@@ -13,7 +13,15 @@ contract BasicToken is ERC20Basic, SafeMath {
 
   mapping(address => uint) balances;
 
-  function transfer(address _to, uint _value) {
+/*
+ * Fix for the ERC20 short address attack  
+ */
+  modifier onlyPayloadSize(uint size) {
+     assert(msg.data.length >= size + 4);
```

Comment:
As you are checking a condition on inputs, shouldn't this rather be a `require()`?